Add D-E lettered article stubs from Glossary (#3821)
ppy/osu-wiki: wiki/Glossary/Nintendo_DS_Games/en.md

---
tags:
  - nintendo games
  - ds games
---

> # Nintendo DS Games

Reviewer: ```suggestion
# Nintendo DS games
```

The **Nintendo DS Games** (or *DS Games* for short) were a set of original rhythm games developed by [iNiS](https://en.wikipedia.org/wiki/INiS) for the [Nintendo DS](https://en.wikipedia.org/wiki/Nintendo_DS) in the early 2000s. The gameplay of these games inspired the creation of the [osu!standard](/wiki/Game_Modes/osu!) [game mode](/wiki/Game_Modes). The DS games' level design also influenced the early years of osu!standard mapping heavily.

> The games referred to are:

Reviewer: ```suggestion
The games referred to are as follows:
```

> - [Osu! Tatakae! Ouendan!](https://en.wikipedia.org/wiki/Osu!_Tatakae!_Ouendan)
> - [Moero! Nekketsu Rhythm Damashii Osu! Tatakae! Ouendan! 2](https://en.wikipedia.org/wiki/Moero!_Nekketsu_Rhythm_Damashii_Osu!_Tatakae!_Ouendan_2)

Reviewer: ```suggestion
- [Osu! Tatakae! Ouendan!](https://en.wikipedia.org/wiki/Osu!_Tatakae!_Ouendan "Wikipedia")
- [Moero! Nekketsu Rhythm Damashii Osu! Tatakae! Ouendan! 2](https://en.wikipedia.org/wiki/Moero!_Nekketsu_Rhythm_Damashii_Osu!_Tatakae!_Ouendan_2 "Wikipedia")
```

---
tags:
  - nintendo games
  - ds games
---

# Nintendo DS Games

> The **Nintendo DS Games** or *DS Games* for short, were a set of rhythm games developed by [iNiS](https://en.wikipedia.org/wiki/INiS) for the [Nintendo DS](https://en.wikipedia.org/wiki/Nintendo_DS) in the early 2000s.
> 
> The gameplay of these games inspired the creation of the [osu!standard](/wiki/Game_Modes/osu!) gamemode, and their level-design influenced the early years of osu!standard mapping heavily.

Reviewer: ```suggestion
In regards to osu! in the osu! wiki, *the* **Nintendo DS Games** (or *DS Games* for short,) were a set of original rhythm games developed by [iNiS](https://en.wikipedia.org/wiki/INiS) for the [Nintendo DS](https://en.wikipedia.org/wiki/Nintendo_DS) in the early 2000s. The gameplay of these games inspired the creation of the [osu!standard](/wiki/Game_Modes/osu!) [game mode](/wiki/Game_Modes). The DS games' level design influenced the early years of osu!standard mapping heavily.
```
Italicized "the" to clarify that, when talked about in regards to osu! in the osu! wiki specifcally, "DS games" refers specifically to the iNiS rhythm games.

Author: clarification is unnecessary since it is implied by being on the osu!wiki that whatever is written is related to osu! in some way or another but i'll try to make it shorter/sweeter and keep the intention the same

The games referred to are:

> - [Osu! Tatakae! Ouendan!](https://en.wikipedia.org/wiki/Osu!_Tatakae!_Ouendan)
> - [Moero! Nekketsu Rhythm Damashii Osu! Tatakae! Ouendan! 2](https://en.wikipedia.org/wiki/Moero!_Nekketsu_Rhythm_Damashii_Osu!_Tatakae!_Ouendan_2)

Reviewer: ```suggestion
- [Osu! Tatakae! Ouendan!](https://en.wikipedia.org/wiki/Osu!_Tatakae!_Ouendan)
  - [Moero! Nekketsu Rhythm Damashii Osu! Tatakae! Ouendan! 2](https://en.wikipedia.org/wiki/Moero!_Nekketsu_Rhythm_Damashii_Osu!_Tatakae!_Ouendan_2)
```

> - [Elite Beat Agents](https://en.wikipedia.org/wiki/Elite_Beat_Agents)

Reviewer: Shouldn't this link to the stub?

Author: whoops

# Nintendo DS Games

> The **Nintendo DS Games** (or *DS Games* for short) were a set of original rhythm games developed by [iNiS](https://en.wikipedia.org/wiki/INiS) for the [Nintendo DS](https://en.wikipedia.org/wiki/Nintendo_DS) in the early 2000s. The gameplay of these games inspired the creation of the [osu!standard](/wiki/Game_Modes/osu!) [game mode](/wiki/Game_Modes). The DS games' level design also influenced the early years of osu!standard mapping heavily.

Reviewer: ```suggestion
The **Nintendo DS Games** (or *DS Games* for short) were a set of original rhythm games developed by [iNiS](https://en.wikipedia.org/wiki/INiS "Wikipedia") for the [Nintendo DS](https://en.wikipedia.org/wiki/Nintendo_DS "Wikipedia") in the early 2000s. The gameplay of these games inspired the creation of the [osu!standard](/wiki/Game_Modes/osu!) [game mode](/wiki/Game_Modes). The DS games' level design also influenced the early years of osu!standard mapping heavily.
```
